components/menu-items/item.js

```diff
@@ -19,10 +25,18 @@ import './style.scss';
 function MenuItemsItem( { className, icon, label, onClick, shortcut, isSelected = false } ) {
 	className = classnames( 'components-menu-items__button', {
 		[ className ]: Boolean( className ),
-		'is-selected': isSelected,
+		'has-icon': Boolean( icon ),
```

Comment:
Same note as at https://github.com/WordPress/gutenberg/pull/5761#discussion_r176813993 . The `Boolean` coercion is redundant and unnecessary.